include/TI/tivx_ext_suppl_data.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2024 The Khronos Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _TIVX_EXT_SUPPL_DATA_H_
+#define _TIVX_EXT_SUPPL_DATA_H_
+
+#include <VX/vx_khr_user_data_object.h>
+#include <VX/vx_khr_supplementary_data.h>
+#include <TI/tivx_obj_desc.h>
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+/* TIOVX ONLY */
```

Comment:
removed. 
We mentioned "TI only" because you can't use the normal extension (vxGetSupplementaryUserDataObject etc.) when you are running in a kernel and a specific interface must be used. (you can't access the vx_reference within a kernel you must rather use the obj descriptor.